### DISCUSSION TITLE
How to integrate this docker with an existing Symfony 3 project?

### DISCUSSION THREAD

**Discussion by jessybreyne** [dunglas/symfony-docker#324] [Q&A] [answered]
I followed this page but without success : https://github.com/dunglas/symfony-docker/blob/main/docs/existing-project.md

I have an error after the `docker-compose` : 
```shell
$ docker-compose build --no-cache --pull

Building php
Sending build context to Docker daemon  1.182GB
Error response from daemon: dockerfile parse error line 51: Unknown flag: link
ERROR: Service 'php' failed to build : Build failed
```

**Comment by maxhelias** [comment, marked as answer]
I think it's your docker engine that's a bit too old

**Reply by jessybreyne** [reply]
Docker version 20.10.16, build 20.10.16-0ubuntu1
docker-compose version 1.29.2, build unknown
Ubuntu 22.10. I have a recent configuration, no?

**Reply by maxhelias** [reply]
`docker compose` have version 2. And use `docker compose` without the `-` since the version 2.
